config.js

```diff
@@ -2,7 +2,8 @@ const one_day = 1440;
 module.exports = {
   title: "ukraine",
   display_title: "Civilian Harm in Ukraine",
-  SERVER_ROOT: "https://ukraine.bellingcat.com/ukraine-server",
```

Comment:
What do you think about using a flag `USE_DEV_API_PROXY` or a check for `NODE_ENV=development` to activate the proxy?

Reply:
If you think the proxy should be enabled by a config prop of course I am fine with that, only I'd enable it by default.

May motivation for this is to simplify the setup so that newcomers can start developing faster. I stumbled myself on it, so I thought it'd make sense to make it work out of the box. 

Where would you set `USE_DEV_API_PROXY`? Tying it to `NODE_ENV=development` probably would not make any difference, because I think the proxy wouldn't run anyways when building/running a production build.